services_app/src/androidTest/java/org/opendatakit/webkitserver/service/OdkWebserverServiceTest.java

```diff
@@ -185,69 +185,130 @@ public void testBindingNRestart() {
         }
     }
 
+
     @Test
     public void testServingHelloWorldHtml() {
-        ODKFileUtils.verifyExternalStorageAvailability();
```

Comment:
Why are you removing this? This is to make sure everything is available.

Reply:
please have a look at the 
```
 
@Before
    public void setUp() throws Exception {
        ODKFileUtils.verifyExternalStorageAvailability();
        ODKFileUtils.assertDirectoryStructure(TestConsts.APPNAME);
        StaticStateManipulator.get().reset();
    }

```
the line   ODKFileUtils.verifyExternalStorageAvailability() is already executed before the test and ensure that everything is available  . i dont find it useful to check the availability again and again.

Reply:
@NiranjanNlc You are correct. The PR review interface did not show those lines of code, and I am moving quickly trying to review many Outreachy things. I am resolving the comment. 

I agree the lines are unnecessary because of the setUp function.